lua/neoclip/sorted_set.lua

```diff
@@ -47,6 +47,16 @@ local insert = function(self, entry, opts)
     end
 end
 
+local edit = function(self, entry, newEntry)
```

Comment:
Would it make more sense to call this `replace`?

Reply:
Yes. I like that name more